.github/workflows/certification.yml

```diff
@@ -159,6 +166,22 @@ jobs:
           echo "$secretName=$value" >> $GITHUB_ENV
         done
 
+    # Authenticate with GCP Workload Identity Pool
+    # Exports GCP ENV Vars:
+    #    - GCP_PROJECT
+    #    - GOOGLE_APPLICATION_CREDENTIALS
+    - id: 'auth'
+      if: matrix.require-gcp-credentials == 'true'
+      name: 'Authenticate to Google Cloud'
+      uses: 'google-github-actions/auth@v1'
+      with:
+        token_format: 'access_token'
+        workload_identity_provider: 'projects/369878874207/locations/global/workloadIdentityPools/contrib-cert-tests-gh-pool/providers/contrib-cert-tests-gh-provider'
+        service_account: 'comp-contrib-wif@dapr-tests.iam.gserviceaccount.com'
```

Comment:
I think these should be defined as a variable in the project. @berndverst and I can help you add that to the repo

Reply:
@ItalyPaleAle  @berndverst 💯 Agreed! I was thinking about these variables:
```
 workload_identity_provider: ${{ secrets.GCP_WIF_PROVIDER_NAME }}
 service_account: ${{ secrets.GCP_WIF_SA_EMAIL }}
```
Does that make sense?

Reply:
Here are the values for the SECRETS vars: 

- `secrets.GCP_WIF_PROVIDER_NAME`=**'projects/369878874207/locations/global/workloadIdentityPools/contrib-cert-tests-gh-pool/providers/contrib-cert-tests-gh-provider'**
- `secrets.GCP_WIF_SA_EMAIL`=**'comp-contrib-wif@dapr-tests.iam.gserviceaccount.com'**